diff -rupN Elixir-0.7.1.orig/elixir/entity.py Elixir-0.7.1/elixir/entity.py
--- Elixir-0.7.1.orig/elixir/entity.py	2013-11-16 01:05:52.249635429 +0100
+++ Elixir-0.7.1/elixir/entity.py	2013-11-16 01:06:56.290894827 +0100
@@ -15,8 +15,9 @@ import sqlalchemy
 from sqlalchemy import Table, Column, Integer, desc, ForeignKey, and_, \
                        ForeignKeyConstraint
 from sqlalchemy.orm import MapperExtension, mapper, object_session, \
-                           EXT_CONTINUE, polymorphic_union, ScopedSession, \
+                           EXT_CONTINUE, polymorphic_union, \
                            ColumnProperty
+from sqlalchemy.orm.scoping import ScopedSession
 from sqlalchemy.sql import ColumnCollection
 
 import elixir
